packages/schematics/src/utils/file.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { addApp } from './file';
```

Comment:
it looks like you have some duplication:

file.ts and fileutils.ts
file.spec.ts and fileutils.spec.ts